build/lib/helm_require.sh

```diff
@@ -45,15 +45,22 @@ spec:
 REQUIRES_CONFIG_FILE=helm/requires-config.yaml
 SEDFILE=${OUTPUT_DIR}/helm/sedfile
 export IMAGE_TAG
-export HELM_REGISTRY
+export HELM_REGISTRY=$(aws ecr-public describe-registries --region us-east-1  --output text --query 'registries[*].registryUri')
 envsubst <helm/sedfile.template >${SEDFILE}
 # Semver requires that our version begin with a digit, so strip the v.
 echo "s,version: v,version: ,g" >>${SEDFILE}
 # Semver requires that we use a + before the git hash to denote build info.
 echo "/^version:/s,-,+," >>${SEDFILE}
 for IMAGE in ${HELM_IMAGE_LIST:-}
 do
-  IMAGE_SHASUM=$(${SCRIPT_ROOT}/image_shasum.sh ${HELM_REGISTRY} ${IMAGE} ${LATEST})
+  if [ "${IMAGE}" == "${HELM_DESTINATION_REPOSITORY}" ]
+  then
+    TAG="${IMAGE_TAG}"
+  else
+    TAG="${LATEST}"
+  fi
+  IMAGE_SHASUM=$(${SCRIPT_ROOT}/image_shasum.sh ${HELM_REGISTRY} ${IMAGE} ${TAG}) ||
```

Comment:
Might want to only push if `$HELM_REGISTRY` isn't blank.

Reply:
The `image_shasum.sh` script will  fail with a parameter not set error